### PR TITLE
Release v1.2.4

### PR DESCRIPTION
**Pull request by jathanism**
Request to merge `develop` into `main` for release v1.2.4.

### CODE REVIEW

File: nautobot/docs/release-notes/version-1.2.md [hunk 1]
```diff
@@ -140,7 +140,24 @@ All models that have `slug` fields now use `AutoSlugField` from the `django-exte
 
 Just as with the UI, the `slug` can still always be explicitly set if desired.
 
-## v1.2.3 (2021-01-07)
+## v1.2.4 (2022-??-??)
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.2.md:143]:
```suggestion
## v1.2.4 (2022-01-13)
```

File: nautobot/docs/release-notes/version-1.2.md [hunk 2]
```diff
@@ -233,6 +250,7 @@ Just as with the UI, the `slug` can still always be explicitly set if desired.
 ### Fixed
 
 - [#453](https://github.com/nautobot/nautobot/issues/453) - Fixed potential `ValueError` when rendering `JobResult` detail view with non-standard `JobResult.data` contents.
+- [#616](https://github.com/nautobot/nautobot/issues/616) - A CustomField with a type of "text" will only validate if the provided value is a string.
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.2.md:253]:
```suggestion
```

File: pyproject.toml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "1.2.3"
+version = "1.2.4-alpha.0"
```
Comment (glennmatthews) [inline pyproject.toml:5]:
```suggestion
version = "1.2.4"
```

**Review by glennmatthews — APPROVED** [review]

File: nautobot/docs/release-notes/version-1.2.md (hunk 2 above)
Comment (jathanism) [reply nautobot/docs/release-notes/version-1.2.md:253]: I'm confused by this? I see #616 as closed.

Comment (jathanism) [reply nautobot/docs/release-notes/version-1.2.md:253]: Nevermind. I see it in there twice.

**Comment by bryanculver** [comment]
LGTM :shipit:

**Review by bryanculver — APPROVED** [review]

**Pull request merged**